src/main/kotlin/no/skatteetaten/aurora/boober/feature/DeploymentConfigFeature.kt

```diff
@@ -123,7 +123,7 @@ class DeploymentConfigFeature() : Feature {
 
                 dc.spec.template.metadata.labels = dc.spec.template.metadata.labels?.addIfNotNull(dcLabels) ?: dcLabels
                 dc.spec.template.metadata.annotations = mutableMapOf(
-                    ANNOTATION_BOOBER_DEPLOYTAG to adc.dockerTag
+                    ANNOTATION_BOOBER_DEPLOYTAG to adc.version
```

Comment:
Får dette noen ringvirkninger i andre systemer? Endrer det visning av releaseto-tag i konsollet?

Reply:
Nei det skal det jo ikke. Dette er jo kun for den som lager taggen. Den sin deploytag bør være det som er spesifisert i version etter min mening

Reply:
Det er gjort noen feil antakelser i konsollet og wallboard ang dette. Det blir fikset i konsollet nå og wallboard senere. 
Jeg oppdaterer her når konsollet er oppdatert

Reply:
Den er nå merget inn og vil ikke bli et problem